src/Type/Definition/BooleanType.php

```diff
@@ -14,33 +13,22 @@
 
 class BooleanType extends ScalarType
 {
-    /** @var string */
     public $name = Type::BOOLEAN;
 
-    /** @var string */
     public $description = 'The `Boolean` scalar type represents `true` or `false`.';
 
     /**
-     * Serialize the given value to a boolean.
+     * Serialize the given value to a bool.
```

Comment:
I'd use `value of bool type` or kept `boolean`

Reply:
Capitalized it, since it refers to the GraphQL type.